packages/xo-web/src/common/intl/messages.js

```diff
@@ -2501,6 +2501,7 @@ const messages = {
   summary: 'Summary',
   wrongNumberOfHosts: 'Wrong number of hosts',
   xostor: 'XOSTOR',
+  xostorCommunity: 'XOSTOR is available in XOA',
```

Comment:
The key doesn't really represent the meaning of the message here. Maybe something like `xostorAvailableInXoa`?